source/wp-content/themes/wporg-news-2021/sass/components/_categories.scss

```
@@ -88,33 +88,131 @@ body.category-community {
 
 	@extend %local-header-lightish;
 
-	.wp-block-post-title,
-	.wp-block-post-date {
-		font-size: var(--wp--preset--font--size--tiny);
+	.wp-site-blocks > .site-content-container:not(.site-header-container):not(.local-header):not(.global-footer):not(.footer-archive) {
+		padding: 0;
+		margin-top: 0;
+
+		@include break-small() {
+			padding-left: var(--wp--custom--alignment--edge-spacing);
+			padding-right: var(--wp--custom--alignment--edge-spacing);
+		}
 	}
 
-	// Alternating blue-1 and blue-2 for non-thumbnail posts
-	li.post:not(.has-post-thumbnail):nth-of-type(even) {
-		background-color: var(--wp--preset--color--blue-1);
-		color: var(--wp--preset--color--white);
-		vertical-align: bottom;
+	.wp-block-post-template {
+		gap: 2px;
 	}
 
-	li.post:not(.has-post-thumbnail):nth-of-type(odd) {
-		background-color: var(--wp--preset--color--blue-2);
-		color: var(--wp--preset--color--white);
-		vertical-align: bottom;
+	.wp-block-post-title {
+		font-size: var(--wp--preset--font-size--huge);
+		line-height: 1.3;
+
+		a:hover,
+		a:focus {
+			color: inherit;
+		}
 	}
 
-	// Featured image only for posts that have one
-	li.post.has-post-thumbnail .wp-block-post-title,
-	li.post.has-post-thumbnail .entry-meta {
-		display: none;
+	.wp-block-post-date {
+		font-size: var(--wp--preset--font-size--small);
 	}
 
-	// B&W featured image
-	li.post.has-post-thumbnail .wp-block-post-featured-image {
-		filter: grayscale(100%);
-		object-fit: cover;
+	li.post {
+		margin: 0;
+
+		@include break-small() {
+			aspect-ratio: 1 / 1;
+		}
+
+		.entry-meta {
+			padding: 24px;
+		}
+
+		&.has-post-thumbnail {
+			.wp-block-template-part,
+			header,
+			figure,
+			figure a,
+			img {
+
+				@include break-small() {
+					height: 100%;
+				}
+			}
+			// B&W featured image
+			.wp-block-post-featured-image {
+				object-fit: cover;
+
+				img {
+					filter: grayscale(100%);
+				}
+			}
+		}
+
+
+		&:not(.has-post-thumbnail) {
+			min-height: 80vh;
+			display: flex;
+			align-items: flex-end;
+
+			@include break-small() {
+				min-height: 0;
+				&:nth-of-type(even):hover,
+				&:nth-of-type(odd):hover {
+					transition: all 0.3s ease-in-out;
+					background-color: black;
+				}
+			}
+
+			// Alternating blue-1 and blue-2 for non-thumbnail posts
+			&:nth-of-type(even) {
+				background-color: var(--wp--preset--color--blue-1);
+				color: var(--wp--preset--color--white);
+				vertical-align: bottom;
+			}
+
+			&:nth-of-type(odd) {
+				background-color: var(--wp--preset--color--blue-2);
+				color: var(--wp--preset--color--white);
+				vertical-align: bottom;
+			}
+		}
+		// Featured image only for posts that have one
+		&.has-post-thumbnail {
+			position: relative;
+
+			@include break-small() {
+				&:hover {
```

This shows up correctly on hover, but not when tabbing through the page with the keyboard. You should be able to use `focus-within` to also show the content when the links are focused, ex:

```suggestion
				&:hover,
				&:focus-within {
```